package.json

```diff
@@ -11,29 +11,55 @@
     "db:migrate": "knex migrate:latest",
     "db:seeds": "knex seed:run",
     "db:setup": "npm run db:migrate && npm run db:seeds",
-    "db:rollback": "knex migrate:rollback"
+    "db:rollback": "knex migrate:rollback",
+    "webdev": "webpack serve  --config config/webpack.client.js --mode development --env development",
+    "dev:build:server": "webpack --config config/webpack.server.js --mode development",
+    "ssr:react": "node --exec babel-node src/server/server.js"
   },
   "author": "Jordan Kelly",
   "license": "ISC",
   "dependencies": {
+    "@loadable/babel-plugin": "^5.13.2",
+    "@loadable/webpack-plugin": "^5.14.0",
+    "babel-loader": "^8.1.0",
```

Comment:
We have quite a few dependencies that should only be dev dependencies. We want our implementation to be as lean as possible. No unnecessary extras.